Cargo.toml

```
@@ -1,21 +1,16 @@
-[workspace]
-members = ["dsp-domain", "dsp-meta", "dsp-meta-cmd", "serde-rdf"]
-resolver = "2"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
-[workspace.package]
-version = "2.0.3"
+[package]
+name = "dsp-meta"
+version = "2.1.0"
```

should be 2.2.0, right?
```suggestion
version = "2.2.0"
```

---

yes, good catch, thanks.